src/types/issues-diff-response.interface.ts

```diff
@@ -3,4 +3,5 @@ import { IIssueResponse } from './issues-response.interface';
 export interface IIssuesDiffResponse {
   issues: IIssueResponse[];
   message?: string;
+  magicLink: string;
```

Comment:
I think this should also be made optional, similar to how it's handled in drawIssuesTable.